### PR TITLE
Update cloud-recording.md

### PR DESCRIPTION
**Pull request by SamChung-MS**
Updated text around Stream recordings

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/cloud-recording.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Learn more about how you can [assign licenses to users in Microsoft 365 or Offic
 
 ### Make sure users have upload video permissions in Microsoft Stream
 
-By default, everyone in the company can create content in Stream, once Stream is enabled and the license is assigned to the user. A Microsoft Stream administrator can [restrict employees for creating content](https://docs.microsoft.com/stream/restrict-uploaders) in Stream. The users who are in this restricted list will not be able to record meetings.
+By default, everyone in the company can create content in Stream, once Stream is enabled and the license is assigned to the user. A Microsoft Stream administrator can [restrict employees for creating content](https://docs.microsoft.com/stream/restrict-uploaders) in Stream. The users who are in this restricted list will still be able to record, however, their recordings will not be able to uploaded to Stream. Instead, these users' recordings will be stored in a temporary storage location where users will be able to download the recording for 21 days. Afterwhich the recording will be deleted.
```
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:66]:
```suggestion
By default, everyone in the company can create content in Stream, once Stream is enabled and the license is assigned to the user. A Microsoft Stream administrator can [restrict employees for creating content](https://docs.microsoft.com/stream/restrict-uploaders) in Stream. The users who are in this restricted list will still be able to record; however, their recordings will not be able to upload to Stream. Instead, these users' recordings will be stored in a temporary storage location where users will be able to download the recording for 21 days. After this period the recording will be deleted.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@tonysmit  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@SamChung-MS - Could you please commit my suggestions. Many thanks.

**Comment by SamChung-MS** [comment]
Comments looked good. Thank you!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
Hello @scanum.  I'm not sure if this PR is for me or Tony?  Either way, it looks like there are conflicting flies.  Can we resolve the conflicts and then I'll review to merge?

**Comment by CarolynRowe** [comment]
@scanum;  Please resolve the conflicts and resubmit for merging.  Thank you.

**Comment by CarolynRowe** [comment]
Closing this as storing meetings recording is now on ODSP.  If further changes are required, please submit another PR.

**Pull request closed without merging**